sbin/common/sbom.sh

```diff
@@ -1,4 +1,32 @@
 #!/bin/bash
+# Build the CycloneDX Java library and app used for SBoM generation
+buildCyclonedxLib() {
+  local javaHome="${1}"
+
+  # Check if JDK-17 is available, download it if it isn't
+  if [ ! -d "${javaHome}" ]; then
+    # Download JDK-17
+    apiUrlTemplate="https://api.adoptium.net/v3/binary/latest/\${JDK_BOOT_VERSION}/\${releaseType}/linux/\${downloadArch}/jdk/hotspot/normal/\${vendor}"
+    apiURL=$(eval echo ${apiUrlTemplate})
+    JDK_BOOT_VERSION="17"
+    releaseType="ga"
+    downloadArch="x64"
+    vendor="adoptium"
+    apiURL="https://api.adoptium.net/v3/binary/latest/${JDK_BOOT_VERSION}/${releaseType}/linux/${downloadArch}/jdk/hotspot/normal/${vendor}"
+    echo "Downloading GA release of boot JDK version ${JDK_BOOT_VERSION} from ${apiURL}"
```

Comment:
You are missing the actual download..
```
curl -L "${apiURL}" | tar xpzf - --strip-components=1 -C "$jdkDir"
```
Set jdkDir variable to be "${CYCLONEDB_DIR}/jdk"
Also add that folder to https://github.com/adoptium/temurin-build/blob/master/.gitignore

Reply:
Then set javaHome in setupAntEnv()